### ISSUE TITLE
Isometric Camera

### ISSUE THREAD

**Issue by jtemo** [runelite/runelite#7414]
**Describe the solution you'd like**
An isometric camera option for GPU rendering!

I figure we can already up the render distance and zoom out a ton, so why not allow the cosmetic change of a switch from perspective to isometric camera?

**Describe alternatives you've considered**
I'unno, there really aren't any. HDHelper/HDScape has shown off something similar, but I wouldn't call that an alternative.., but after seeing a few videos of a different launcher with this option, I'm in love with the idea!

**Additional context**
https://streamable.com/y7uv9
This video shows off what I'm talking about perfectly.

**Comment by Kevinclon** [comment]
OMG I NEED IT

**Comment by formatme** [comment]
am i missing something or??? you can just put your camera around the same angle and it will look the same?

**Comment by jtemo** [comment]
@formatme yes, but it's understandable. The differences are certainly subtle! The core difference is how things scale and because of this visibly appear: Imagine you're looking at a perfectly square piece of paper from a 45 degree angle, like it was on a table in front of you.

From a perspective camera at this angle (the way your eyes work!), the closer end of the paper appears visibly wider, and the further end visibly shorter. The side lines appear diagonal. The outline of the paper from this angle form a trapezoid. If the side lines went on forever, they would appear to meet at something called a vanishing point. This is perspective! The vanishing point is key to the difference!

From an isometric camera, our paper at this angle would now look like a rectangle. As you angle your camera down, the top and bottom lines get closer to each other, but are still of the same length. The sides of the paper in this camera would appear parallel, and have no vanishing point. An infinitely tall piece of paper at this camera would still look the same width, no matter what.

I would suggest looking up some pictures for reference comparing the two, it'll help describe the phenomenon much better than my text descriptions ever could. Hope it clears things up regardless!
